### ISSUE TITLE
edit 10man CFG

### ISSUE THREAD

**Issue by Firefly1029** [splewis/csgo-pug-setup#274]
I just want to be able to change the default .10man cfg to enable knife round. I haven't seen a cfg to do this any where.

**Comment by splewis** [comment]
> I just want to be able to change the default .10man cfg to enable knife round. I haven't seen a cfg to do this any where

You can't (it looks like the plugin just uses whatever the last game was setup as: https://github.com/splewis/csgo-pug-setup/blob/12853e4d8c8e269402aa3d03907a110d6cc885f8/scripting/pugsetup.sp#L832-L833). 

I recommend just using the .setup command instead of that command.

> EDIT: also a way to input custom team pick orders or even add the option to select team pick order in the setup menu

Highly unlikely to happen. Pull requests are welcome though.

**Comment by borzaka** [comment]
Edit the [setupoptions.cfg](https://github.com/splewis/csgo-pug-setup/blob/master/configs/pugsetup/setupoptions.cfg) and change `kniferound`'s `default` to `1`

And if you `enable sm_pugsetup_autosetup` (plus optionally `sm_pugsetup_force_defaults`), your game will be autosetup with the default options and kniferound enabled.

And if you don't want to mess with config files, you can use `.setup` to setup your match every time as @splewis said.

**Comment by bboychris168** [comment]
@borzaka would you perhaps know how to setup random captains? instead of "assigning captains"?

**Comment by borzaka** [comment]
There is a command for that: `.rand`/`!rand` for randomly selecting captains.

Or you can edit your config in `cfg/sourcemod/pugsetup.cfg` and change `sm_pugsetup_auto_randomize_captains` to `1`

> (default 0) - When games are using captains, should they be automatically randomionce? Note you can still manually set them use .rand/!rand to redo the randomization

**Comment by bboychris168** [comment]
@borzaka would you also perhaps know if i can disable the damage on console and show it after a round ends?

**Comment by borzaka** [comment]
Sure. To disable the damage feed in the console, use this command:
`sv_damage_print_enable 0`

To show the damage done/taken from players on round ends, use the [pugsetup_damageprinter](https://github.com/splewis/csgo-pug-setup#pugsetup_damageprinter) plugin.

**Comment by bboychris168** [comment]
Yeah sorry im confused on where i should be placing that command? Yeh i figured out how to enable the damage printer.

**Comment by bboychris168** [comment]
@borzaka

**Comment by borzaka** [comment]
You don't have to @ me, and you can edit your comment, no need to write a new one with zero information in it.

Put that command in your server.cfg. Or you can add to the plugin's live.cfg, or whatever config you use.

**Comment by bboychris168** [comment]
Sorry about that. But one very last thing that i also tried figuring out today but had issues doing so. Am i able to disable the ".setup" command once i configure to the settings i want? Ive had users in my csgo channel changing to unwanted settings. Ive up my admin immunities but nothing seems to stop the ".setup" command from getting unwanted changes from the users in the server.

**Comment by borzaka** [comment]
You don't have to disable the `.setup` command, just limit who can access it.
Modify the commands permissions in the `permissions.cfg`

**Comment by bboychris168** [comment]
![permissions folder](https://user-images.githubusercontent.com/43663480/57191625-11a94300-6f6b-11e9-9452-8b09232a9b19.jpg)
I have changed the setup to admin but i did a test today and people can still use the .setup command.

**Comment by borzaka** [comment]
Have you restarted your server after editing the permissions?

If so, no worries, everyone can type `.setup` to see the settings, but only admins can change it.

**Comment by chihebloussif** [comment]
> Have you restarted your server after editing the permissions?
> 
> If so, no worries, everyone can type `.setup` to see the settings, but only admins can change it.

can i set the sv_damage_print_enable 0 to default ????

**Comment by borzaka** [comment]
@chihebloussif I don't understand your question. Also, wrong issue. This wasn't the topic here. Open a new issue if you have problems.

**Comment by chihebloussif** [comment]
> @borzaka would you also perhaps know if i can disable the damage on console and show it after a round ends? 

This is the orignal topic i was asking for , but i arrived to a solution to disable the damage feedback from the console

Solution : add sv_damage_print_enable 0
Ti live.cfg and knife.cfg in pugsetup config commands line then activate the damage_printer plugins from from disabled folder and everything work like a charm
